dotcom-rendering/src/components/InteractiveBlockComponent.stories.tsx

```diff
@@ -132,8 +131,11 @@ export const WithCaption = () => {
 	);
 };
 WithCaption.storyName = 'with caption';
+WithCaption.decorators = [
+	splitTheme([defaultFormat], { orientation: 'vertical' }),
+];
 
-export const NonBootJs = () => {
+export const NonBootJs = ({ format }: StoryProps) => {
```

Comment:
If you so desire, this gives slightly better intellisense:

```suggestion
export const NonBootJs: StoryObj = ({ format }: StoryProps) => {
```